test/integration-test/Cargo.toml

```diff
@@ -28,6 +28,7 @@ test-case = { workspace = true }
 test-log = { workspace = true, features = ["log"] }
 tokio = { workspace = true, features = ["macros", "rt-multi-thread", "time"] }
 xdpilone = { workspace = true }
+nix = { workspace = true, features = ["process"] }
```

Comment:
You would add any other features that were required from `nix` here.
I assume you needed something from the default featureset given the change in the main `Cargo.toml`.